packages/custom-webpack/src/schema.ext.json

```diff
@@ -7,7 +7,8 @@
         "path": {"type": "string", "description": "Path to the custom webpack configuration file"},
         "mergeStrategies": {"type": "object", "description": "Merge strategies per webpack config field"},
         "replaceDuplicatePlugins": {"type": "boolean", "description": "Flag that indicates whether to replace duplicate webpack plugins or not"}
-      }
+      },
+      "default": null
```

Comment:
I'd rather make it `"default": {}`, this way the integration is flawless and doesn't require any additional change nor unit tests.